packages/flutter/lib/src/material/drawer.dart

```diff
@@ -100,13 +103,23 @@ class Drawer extends StatelessWidget {
   /// {@macro flutter.widgets.child}
   final Widget child;
 
+  /// A name to announce in accessibility frameworks when opening this drawer.
+  ///
+  /// The name is used in Android TalkBack but ignored in iOS.
+  ///
+  /// See also:
+  ///
+  ///   * [Semantics.route]
+  final String semanticValue;
```

Comment:
Is it intentional that it is called "semanticValue" here and "semanticName" in the AlertDialog? Could we unify these?

Also - and I know that naming is hard - could we find a more descriptive name for the property? Maybe something that tells the user that this is used as the route's name for semantics announcements?

Reply:
could also just go with routeName so that it is consistent with routes ... but I worry that will create confusion since this is not an actual route